src/DFTK.jl

```diff
@@ -18,6 +18,9 @@ using Random
 using ChainRulesCore
 using PrecompileTools
 
+using AtomsBase
+using AtomsCalculators
```

Comment:
`using AtomsBase` should already be done somewhere, so not needed. The `using AtomsCalculators` you best do just in the file implementing the interface.